plugin/ochttp/client_stats.go

```diff
@@ -61,6 +61,7 @@ func (t statsTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 		track.end()
 	} else {
 		track.statusCode = resp.StatusCode
+		track.respContentLength = resp.ContentLength
```

Comment:
At line 52 we do something similar. Not sure I understand. Why is that not enough?